build/packages/rpm/server/SPECS/pmm-update.spec

```diff
@@ -5,7 +5,7 @@
 %define _binaries_in_noarch_packages_terminate_build   0
 %define _unpackaged_files_terminate_build   0
 
-%global repo            pmm-update
+%global repo            pmm
```

Comment:
let's change value for https://github.com/percona/pmm/pull/1485/files#diff-8dc435ece712d63bec31a2bf14c0d35906219b779647104244ad742c38cd7323R20 to have proper name

Reply:
and let's add a new record into changelog